packages/neo-one-monitor/src/BrowserMonitor.js

```diff
@@ -30,30 +55,193 @@ class BrowserGauge extends BaseMetric implements Gauge {
 }
 
 class BrowserHistogram extends BaseMetric implements Histogram {
-  // eslint-disable-next-line
-  observe(countOrLabels?: number | RawLabels, count?: number): void {}
+  observe(countOrLabels?: number | RawLabels, count?: number): void {
+    this.values.push({ countOrLabels, count });
+  }
 }
 
 class BrowserSummary extends BaseMetric implements Summary {
   // eslint-disable-next-line
   observe(countOrLabels?: number | RawLabels, count?: number): void {}
 }
 
-class BrowserMetricsFactory implements MetricsFactory {
+interface MetricsFactoryCollect extends MetricsFactory {
+  collect(): Object;
+}
+
+type MetricCollection = {|
+  counters: Array<BaseMetric>,
+  gauges: Array<BaseMetric>,
+  histograms: Array<BaseMetric>,
+  summaries: Array<BaseMetric>,
+|};
+
+class BrowserMetricsFactory implements MetricsFactoryCollect {
```

Comment:
Move this into it's own file and leave the old factory and metrics how they were. I.e. this file should basically have no changes.

Reply:
The reason being is we shouldn't force all consumers to use the metrics that are collected and reported - instead they can use the default no-op implementation.